scripts/populateDb.ts

```diff
@@ -10,30 +10,86 @@ import Team from '../models/team';
 import Score from '../models/scores';
 import Schedule from '../models/schedule';
 
+import { parse } from 'ts-command-line-args';
+
 dotenvConfig();
 
-const { NUM_HACKERS, NUM_JUDGES, JUDGING_LENGTH, NUM_ROOMS, START_TIME_STAMP, EMAIL, USER_TYPE } = process.env;
-
-if (!(NUM_HACKERS && NUM_JUDGES && JUDGING_LENGTH && NUM_ROOMS && START_TIME_STAMP && EMAIL && USER_TYPE)) {
-	console.log('Usage:');
-	console.log(
-		'Please log into Witness using a fresh database. Then, make sure that the following variables are set in your .env:'
-	);
-	console.log('NUM_HACKERS: integer number of hackers to generate');
-	console.log('NUM_JUDGES: integer number of judges to generate');
-	console.log('NUM_ROOMS: integer number of zoom rooms URLs to generate');
-	console.log('JUDGING_LENGTH: length in milliseconds of a judging period');
-	console.log('START_TIME_STAMP: unix timestamp for when "judging" will begin');
-	console.log('EMAIL: email address (case sensitive) of the user you signed in as in the first step.');
-	console.log(
-		'USER_TYPE: one of JUDGE or HACKER or ORGANIZER. Set this to whatever you want your user to be assigned as.'
-	);
-	console.log('================================================================');
-	console.log(
-		'Run the script with DATABASE_URL="..." ts-node <path-to-script>/populateDb.ts where ... is your DEVELOPMENT database connection string.'
-	);
-	process.exit(0);
+interface Arguments {
+	numHackers?: number;
+	numJudges?: number;
+	judgingLength?: number;
+	numRooms?: number;
+	startTimeStamp?: number;
+	email?: string;
+	userType?: string;
+	databaseUrl: string;
+	help?: boolean;
 }
+
+export const args = {
+	// Defaults
+	...{
+		numHackers: 500,
+		numJudges: 20,
+		judgingLength: 10 * 1000,
+		numRooms: 5,
+		startTimeStamp: Date.now() + 60 * 1000,
+		userType: 'HACKER',
+	},
+	...parse<Arguments>(
+		{
+			databaseUrl: { type: String, alias: 'D', description: '***REQUIRED***: Database connection string' },
```

Comment:
should we not use the db url in the env by default?

Reply:
The rationale with this (and the previous version as well) is because this is pretty much a safety check. Prevents people from doing something by accident if they're working off of a different DB in their env.

Reply:
okay that is valid, can we have that documented tho